### PR TITLE
proposal: refine how we define PDA paths for some account types

### PR DESCRIPTION
**Pull request by chrismaree**
this PR refines how we define PDAs in some places to make them more consistently dependent on the seed, rather than state

### CODE REVIEW

File: programs/svm-spoke/src/event.rs [hunk 1]
```diff
@@ -121,7 +121,6 @@ pub struct ClaimedRelayerRefund {
     pub refund_address: Pubkey,
 }
 
-// TODO: update the name of this to EmergencyDeletedRootBundle and in EVM.
```
Comment (Reinis-FRP) [inline programs/svm-spoke/src/event.rs:124]: we have not fixed this in EVM

File: programs/svm-spoke/src/instructions/handle_receive_message.rs [hunk 2]
```diff
@@ -11,9 +11,6 @@ use crate::{
     State,
 };
 
-//TODO: we have inconsistent imports in this file, in some places referencing from source crates (SvmSpoke::id)
```
Comment (Reinis-FRP) [inline programs/svm-spoke/src/instructions/handle_receive_message.rs:14]: we have not addressed this

File: programs/svm-spoke/src/state/state.rs [hunk 3]
```diff
@@ -12,7 +12,7 @@ pub struct State {
     pub current_time: u32,              // Only used in testable mode, else set to 0 on mainnet.
     pub remote_domain: u32,             // CCTP domain for Mainnet Ethereum.
     pub cross_domain_admin: Pubkey,     // HubPool on Mainnet Ethereum.
-    pub root_bundle_id: u32,            // TODO rename to next_root_bundle_id
+    pub root_bundle_id: u32,            // Tracks the current root bundle id.
```
Comment (Reinis-FRP) [inline programs/svm-spoke/src/state/state.rs:15]: but it is not the current root bundle id. It is next one, thus we had comment to rename this prop

**Review by Reinis-FRP — APPROVED** [review]
Using state.seed seems more consistent. though there are some comments we delete that are not yet addressed.

File: programs/svm-spoke/src/state/state.rs (hunk 3 above)
Comment (chrismaree) [reply programs/svm-spoke/src/state/state.rs]: I chose to just update the comment accordingly.

File: programs/svm-spoke/src/instructions/handle_receive_message.rs (hunk 2 above)
Comment (chrismaree) [reply programs/svm-spoke/src/instructions/handle_receive_message.rs:14]: right but the thing it was referencing for some reason could not be refactored like other imports.

File: programs/svm-spoke/src/event.rs (hunk 1 above)
Comment (chrismaree) [reply programs/svm-spoke/src/event.rs:124]: done here: https://github.com/across-protocol/contracts/pull/760

**Pull request merged**